Events/2021-11-rse-hpc-workshop.md

```diff
@@ -0,0 +1,54 @@
+# Research Software Engineers in HPC: Creating Community, Building Careers, Addressing Challenges (RSE-HPC-2021)
+- Date: August 6, 2021
+- Location: St. Louis, MO and virtual
+- Event Website: https://us-rse.org/rse-hpc-2021/
+- Organizers: US Research Software Engineer Association
+			   
+<!-- deck text start -->
+The RSE in HPC workshop will be held in conjunction with SC21, as a hybrid (on-site and virtual) event.
+<!-- deck text end -->
+
+Event Information | Details
+:--- | :---			   
+Event Name | [Research Software Engineers in HPC: Creating Community, Building Careers, Addressing Challenges (RSE-HPC-2021)](https://us-rse.org/rse-hpc-2021/)
+Event Date | November 15, 2021
+Website | https://us-rse.org/rse-hpc-2021/
+**Timeline** |
+Submission Deadline | Friday, August 6, 2021 - NO EXTENSIONS
+Notifications Sent | Friday, September 3, 2021
+Program Finalized | Friday, September 17, 2021
+Workshop Date | Monday, November 15, 2021
+
+Research software engineers (RSEs) are critical to the impact of HPC, data science, and the larger scientific community. They have existed for decades, though often not under that name. However, the past few years have seen the development of the RSE concept, common job titles, and career paths; the creation of professional networks to connect RSEs; and the emergence of RSE groups at universities, national laboratories, and industry.
+
+This workshop will bring together RSEs and allies involved in HPC, from all over the world, to grow the RSE community by establishing and strengthening professional networks of current RSEs and RSE leaders. We’ll discuss the current activities and plans of national RSE organizations, discuss the needs of RSEs and RSE groups, and write a report on ways RSE organizations can help address these.
+
+Most of the workshop time will be spent in interactive discussions, preceded by a keynote address and several shorter talks to establish context.
+
+## Submissions
+
+We encourage prospective participants to submit position papers (limit of 2 pages, not counting references; no format prescribed) on topics related to RSE issues. We particularly encourage submissions on the keynote topic:
+* Do RSEs need to change for exascale?
```

Comment:
```suggestion
* Do RSEs need to change for exascale?  
```

Reply:
I think the extra spaces will end the list and make the next line start a new paragraph